templates/chatbot/template.yaml

```diff
@@ -534,7 +595,6 @@ spec:
       input:
         repoUrl: ${{ parameters.githubServer }}?owner=${{ parameters.repoOwner }}&repo=${{ parameters.repoName }}
         branchName: trigger-pipeline
-        gitCommitMessage: trigger pipeline build
```

Comment:
this should be `commitMessage`

Reply:
Nice! Addressed here [ee451d0](https://github.com/redhat-ai-dev/ai-lab-template/pull/93/commits/ee451d0e8f6aa007f97aa7ef498b153aecc89cbd)